include/nds/arm9/image.h

```diff
@@ -26,9 +26,9 @@
 	\brief An image abstraction for working with image data.
 
 <div class="fileHeader">
-Image data pointers must be allocated using malloc as the conversion
-rutiens will free the pointers and allocate new data.  As such any loader
-implemented utilizing this structure must use malloc() to allocate the image
+Image data buffers must be allocated using malloc(), rather than pointing to stack data, as the conversion
+routiens will free() the argument's image buffer and allocate a new block for the replacement data.  
+As such any loader implemented utilizing this structure must use malloc() to allocate the image
 pointer data.
```

Comment:
Replace "pointer data" with "buffer" to be consistent with terminology used earlier.